curriculum/challenges/english/25-front-end-development/lab-factorial-calculator/66c07238b01053abaf812065.md

```diff
@@ -0,0 +1,48 @@
+---
+id: 66c07238b01053abaf812065
+title: Build a Factorial Calculator
+challengeType: 14
+dashedName: build-a-factorial-calculator
+---
+
+# --description--
+
+**Objective:** Fulfill the user stories below and get all the tests to pass to complete the lab. 
+
+**User Stories:**
+
+1. You should ask the user for a number using a prompt window and store the number in a variable called `inputNumber`.
```

Comment:
I think this should be simplified to ask for the prompt method instead of prompt window

```suggestion
1. You should ask the user for a number using the `prompt` method and store the number in a variable called `inputNumber`.
```